.github/workflows/ci.yml

```diff
@@ -48,3 +48,14 @@ jobs:
             unlist()
           warnings_found <- any(grepl("expectation_warning", expectations))
           quit(status = warnings_found)
+      - name: Spell Check
+        if : always()
+        shell: Rscript {0}
+        run: |
+          spell_check <- spelling::spell_check_package(use_wordlist = TRUE)
+          cli::cli_alert_warning("There are {nrow(spell_check)} spelling error{?s}.")
+          if (nrow(spell_check) > 0) {
+            print(spell_check)
+          }
+          quit(status = nrow(spell_check) > 0)
+          
```

Comment:
Avoid trailing whitespace to keep diffs small and meaningful (configure your editor to do it automatically for you).
```suggestion
```

Reply:
Extra line should be removed now.